src/app/core/header/header.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrl: './header.component.css'
+})
+export class HeaderComponent {
+  headerText: string = 'your profile';
```

Comment:
Wouldn't it be simpler to put the HTML and CSS of this header wherever it is needed? The text of this header is fixed and as such it can be used in only one page.

Reply:
Yes, this should be an @Input